crates/typst-library/src/text/mod.rs

```diff
@@ -364,6 +364,17 @@ pub struct TextElem {
     #[resolve]
     pub hyphenate: Hyphenate,
 
+    /// The "cost" of hyphenation when laying out text.
+    /// A higher cost means the layout engine will hyphenate less often.
+    ///
+    /// When justifying text, a higher hyphenation cost can result in more variation
+    /// in justification spacing in order to avoid hyphenation.
+    ///
+    /// The default cost is an acceptable balance, but some may find that it hyphenates too eagerly.
```

Comment:
Maybe mention that the default is `0.5` here.

Reply:
I figured since the docs show the default it's not necessary to repeat it

Reply:
Oh never mind then!